test/controllers/restaurants_controller_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class RestaurantsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
Good job writing tests! 🙌

Reply:
Thank you but I didnt write these (automatically created)